sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisTest.scala

```diff
@@ -53,16 +53,18 @@ trait AnalysisTest extends PlanTest {
   }
 
   protected def getAnalyzer(caseSensitive: Boolean) = {
-    if (caseSensitive) caseSensitiveAnalyzer else caseInsensitiveAnalyzer
+    makeAnalyzer(caseSensitive)
```

Comment:
why do we need to change it?

Reply:
yes, it's duplicated, since caseSensitiveAnalyzer and caseInsensitiveAnalyzer are already change from val to def. will revert.